tests/serialization/test_dag_serialization.py

```diff
@@ -887,6 +887,7 @@ def test_no_new_fields_added_to_base_operator(self):
             'executor_config': {},
             'inlets': [],
             'label': '10',
+            'max_active_tis_per_dag': None,
```

Comment:
What's the purpose of this test BTW? We had another PR About it and I was not sure why we have the list here.

Reply:
This list just makes sure that if new fields are added to the base operator we add it here, in case it needs any special treatment for DAG Serialization (helpful if they are dicts and lists)